src/main/webapp/app/entities/exercise.model.ts

```diff
@@ -120,6 +120,7 @@ export abstract class Exercise implements BaseEntity {
 
     // helper attributes
     public secondCorrectionEnabled = false;
+    public isTogglingSecondCorrection?: boolean;
```

Comment:
With this change you are encoding UI/server state information inside the domain model. I think that should definitely be avoided if possible. Is there an alternative solution?

Reply:
Good point. Yes, I agree. Sure, there is a better suitable solution :+1: (let me think about it :D)